src/cloud-functions.ts

```diff
@@ -314,23 +301,7 @@ export function makeCloudFunction<EventData>({
       });
   };
 
-  if (process.env.X_GOOGLE_NEW_FUNCTION_SIGNATURE === 'true') {
-    cloudFunction = cloudFunctionNewSignature;
-  } else {
-    cloudFunction = (raw: Event | LegacyEvent) => {
-      let context;
-      // In Node 6 runtime, function called with single event param
-      let data = _.get(raw, 'data');
-      if (isEvent(raw)) {
-        // new eventflow v1beta2 format
-        context = _.cloneDeep(raw.context);
-      } else {
-        // eventflow v1beta1 format
-        context = _.omit(raw, 'data');
-      }
-      return cloudFunctionNewSignature(data, context);
-    };
-  }
+  cloudFunction = cloudFunctionNewSignature;
```

Comment:
can remove the let cloudFunction above and do const here, or just use cloudFunctionNewSignature below

Reply:
Good catch - changed to use cloudFunction everywhere.